app/controllers/users_controller.rb

```diff
@@ -11,13 +12,20 @@ def show; end
 
   def new
     @user = User.new
+    @user.build_profile
+    @user.profile.build_contact
```

Comment:
The traditional Rails way for this is [nested attributes](https://rubyplus.com/articles/3681-Complex-Forms-in-Rails-5), but it has its downsides as well. Not saying you should change it in this PR, just something to read up on :)